bindings/CXX11/CMakeLists.txt

```diff
@@ -66,13 +66,13 @@ install(TARGETS adios2_cxx11 ${maybe_adios2_cxx11_mpi} EXPORT adios2CXX11Exports
 )
 
 install(
-  FILES adios2.h 
+  FILES adios2.h
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
   COMPONENT adios2_cxx11-development
 )
 
 install(
-  FILES adios2/cxx11/fstream/ADIOS2fstream.h 
```

Comment:
rogue trailing space

Reply:
I did rebase and I just got rid of the formatting changes -- that was just my editor removing trailing spaces, but I agree that it shouldn't be mixed in with actual changes.